Makefile

```diff
@@ -204,12 +204,16 @@ endif
 # Dependencies
 # ------------
 
-.PHONY: verify-deps
-verify-deps:
+.PHONY: verify-go-deps
+verify-go-deps:
 	go mod verify
 	go mod tidy
 	@git diff --exit-code -- go.mod go.sum
 
+.PHONY: verify-src-deps
+verify-src-deps: $(SRC_DEPS)
```

Comment:
There is no benefit of checking all files as it hides original intention. 
Let's just focus on verifying the generated files `pkg/api/generated/openapi/zz_generated.openapi.go`, `pkg/scraper/types_easyjson.go`.